develop-docs/sdk/miscellaneous/unified-api/index.mdx

```diff
@@ -216,6 +216,8 @@ Sentry.configureScope((scope) =>
 
 - `scope.set_extras(extras)`: Sets an object with key/value pairs, convenience function instead of multiple `set_extra` calls. As with `set_extra` this is considered deprecated functionality.
 
+- `scope.set_flag(key, value)`: Sets the flag to a string value, overwriting a potential previous value. A maximum of 100 flags can be retained by the SDK. Entries can not be removed except by expiration.
```

Comment:
I'd leave the max value out of this or phrase it to mean that the "current default" or something? We might want to make this configurable or change the default and I'm afraid we won't find all sports in the docs where we wrote 100